x-pack/plugins/monitoring/public/directives/main/index.html

```diff
@@ -210,7 +210,7 @@
           >
           </a>
           <a
-            ng-if="monitoringMain.instance"
+            ng-if="monitoringMain.instance && monitoringMain.pipelineCount > 0 && monitoringMain.shouldDisplayPipelineNav === true"
```

Comment:
@ycombinator it's possible that the second condition in this `ng-if` is redundant now, but I did not test that.